include/rdma/fabric.h

```diff
@@ -41,6 +41,10 @@
 extern "C" {
 #endif
 
+#if defined(_WIN32)
+#include <BaseTsd.h>
+	typedef SSIZE_T ssize_t;
+#endif
```

Comment:
nit - I'd move this even higher, near the other include commands, outside of the extern "C"